www/cgi-bin/dump

```diff
@@ -95,7 +101,7 @@ start)
 	for i in 1 2 3
 	do
 		push_event $EVENT_ID UPLOAD "Uploading capture for ${PATHS[2]} to CloudShark ($i try)..."
-		JSON_STATUS=`curl --cacert /usr/lib/cdns/cacert.pem -H "Origin: https://appliance.cloudshark.org" -F "capture[upload]=@${DUMP_FILE}.0" https://www.cloudshark.org/captures.json`
+		JSON_STATUS=`curl --cacert /usr/lib/cdns/cacert.pem --upload-file ${DUMP_FILE}.0 https://www.cloudshark.org/api/v1/${CLOUDSHARK_API_TOKEN}/upload`
```

Comment:
Took a look, and nowhere else in that file is $DUMP_FILE surrounded by double quotes. While I agree it would be safer to do that, this change matches existing usage. Quoting it should probably be a separate commit outside the scope of this pull request.